docs/source/index.rst

```diff
@@ -10,10 +10,10 @@
                     <h3 class="ui header">Friendly modelling API</h3>
                     <p>PyMC3 allows you to write down models using an intuitive syntax to describe a data generating
                         process.</p>
-                    <h3 class="ui header">Cutting edge algorithms</h3>
+                    <h3 class="ui header">Cutting edge algorithms and modules</h3>
```

Comment:
models?

Reply:
Even with `models` I'm not sure what that is supposed to mean.

Reply:
Yeah I just want to express that gp is not really an algorithm - at least not an inference algorithm

Reply:
Cutting edge inference algorithms and model building blocks?